dask_sql/physical/rel/base.py

```diff
@@ -104,7 +104,13 @@ def fix_dtype_to_row_type(dc: DataContainer, row_type: "RelDataType"):
         }
 
         for field_name, field_type in field_types.items():
-            expected_type = sql_to_python_type(field_type.getSqlType())
+            sql_type = field_type.getSqlType()
+            sql_type_args = tuple()
+
+            if str(sql_type) == "SqlTypeName.DECIMAL":
+                sql_type_args = field_type.getDataType().getPrecisionScale()
+
+            expected_type = sql_to_python_type(sql_type, *sql_type_args)
```

Comment:
Does this pattern seem okay for optionally deciding if we need to push creation args to the Python type we're creating? Would like to make sure there's not a better way to do this before applying it in the other places where we call `sql_to_python_type`

Reply:
Yeah, this seems reasonable especially since Decimal right now is the only type we have where there is additional information attached to the type.